plugins/legacy-plugin-chart-sankey/src/Sankey.js

```diff
@@ -22,6 +22,8 @@ import d3 from 'd3';
 import PropTypes from 'prop-types';
 import { sankey as d3Sankey } from 'd3-sankey';
 import { getNumberFormatter, NumberFormats, CategoricalColorNamespace } from '@superset-ui/core';
+import './Sankey.css';
```

Comment:
Let's use styled components instead of adding new CSS files.

E.g. https://github.com/apache-superset/superset-ui/blob/482da217938bcbd70db8b79117abacf00f8037e2/plugins/legacy-plugin-chart-parallel-coordinates/src/ReactParallelCoordinates.jsx#L36